sdk/ai/Azure.AI.Inference/api/Azure.AI.Inference.net8.0.cs

```diff
@@ -2,8 +2,8 @@ namespace Azure.AI.Inference
 {
     public static partial class AIInferenceExtensions
     {
-        public static Azure.AI.Inference.ChatCompletionsClient GetChatCompletionsClient(this System.ClientModel.Primitives.ConnectionProvider provider) { throw null; }
-        public static Azure.AI.Inference.EmbeddingsClient GetEmbeddingsClient(this System.ClientModel.Primitives.ConnectionProvider provider) { throw null; }
+        public static Azure.AI.Inference.ChatCompletionsClient GetChatCompletionsClient(this System.ClientModel.Primitives.ConnectionProvider provider, string? connectionName = null, Azure.AI.Inference.AzureAIInferenceClientOptions? options = null) { throw null; }
```

Comment:
We already have a concept of connectionId in Connectionprovider,GetConnections. We cannot also add connectionName. It will be confusing.